localstack/services/events/target.py

```diff
@@ -0,0 +1,304 @@
+from __future__ import annotations
```

Comment:
I think it is only problematic in the provider, reflected that in the comments, but also removed in targets since it is also here a leftover from previous iteration removed in: https://github.com/localstack/localstack/pull/10613/commits/122fcc95f00a70cb36782c1de2b559aa9d1db7c3